app/editor/src/store/slices/content/contentSlice.ts

```diff
@@ -36,6 +36,11 @@ export const initialContentState: IContentState = {
     searchTerm: '',
   },
   filterMorningReports: defaultMorningReportsFilter(),
+  filterMorningReportAdvanced: {
```

Comment:
They were originally sharing an advanced filter which was causing all kinds of issues.